middleware/downloadcsv.js

```diff
@@ -1,18 +1,56 @@
 const { parse, transforms: { unwind } } = require("json2csv")
+const { parseQuestionText } = require("../utils/textUtils")
+
+const get_qcode = (row) => {
+  if (row.answers.options && row.answers.options.q_code) {
+    return row.answers.options.q_code
+  }
+  if (row.answers.options && row.answers.options.qCode) {
+    return row.answers.options.qCode
+  }
+  if (row.answers.q_code) {
+    return row.answers.q_code
+  }
+  if (row.answers.qCode) {
+    return row.answers.secondaryQCode ? `${row.answers.qCode} / ${row.answers.secondaryQCode}` : row.answers.qCode
```

Comment:
Just checking - is this line still needed or were both to be reverted back?

Reply:
that is still needed.